chia/cmds/db_upgrade_func.py

```diff
@@ -314,26 +323,39 @@ async def convert_v1_to_v2(in_path: Path, out_path: Path) -> None:
                     commit_in -= 1
                     if commit_in == 0:
                         commit_in = COIN_COMMIT_RATE
-                        await out_db.executemany("INSERT INTO coin_record VALUES(?, ?, ?, ?, ?, ?, ?, ?)", coin_values)
-                        await out_db.commit()
-                        await out_db.execute("begin transaction")
+                        out_db.executemany("INSERT INTO coin_record VALUES(?, ?, ?, ?, ?, ?, ?, ?)", coin_values)
+                        out_db.commit()
+                        out_db.execute("begin transaction")
                         coin_values = []
                         end_time = time()
                         rate = COIN_COMMIT_RATE / (end_time - start_time)
                         start_time = end_time
 
-            await out_db.executemany("INSERT INTO coin_record VALUES(?, ?, ?, ?, ?, ?, ?, ?)", coin_values)
-            await out_db.commit()
+            out_db.executemany("INSERT INTO coin_record VALUES(?, ?, ?, ?, ?, ?, ?, ?)", coin_values)
+            out_db.commit()
             end_time = time()
             print(f"\r      {end_time - coin_start_time:.2f} seconds                             ")
 
             print("[5/5] build indices")
             index_start_time = time()
             print("      block store")
-            await BlockStore.create(DBWrapper(out_db, db_version=2))
+            out_db.execute("CREATE INDEX height on full_blocks(height)")
```

Comment:
I notice this block is new, but it seems to be all indexes and hints, so no worries.

Reply:
right. The existing store classes, `BlockStore`, `CoinStore` and `HintStore` use the `DBWrapper` class to access the database. They specifically use the `aiosqlite` interface to the database, which means we can't use those classes when writing to the DB with the (synchronous) `sqlite3` API.

Reply:
The only reason we created those classes here though, was to create the indices. So, those statements were copy-pasted out of those classes.